go.mod

```diff
@@ -5,5 +5,6 @@ go 1.12
 require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/xeipuuv/gojsonschema v1.2.0
-	neilpa.me/go-x v0.1.0
+
+	sigs.k8s.io/yaml v1.2.0
```

Comment:
Is there a reason to use this instead of the original [ghodss/yaml](https://github.com/ghodss/yaml) which the k8s version is a fork of?

Reply:
Good catch. No, this was only pulled in because, as you saw, much of the code was copied from helm originally and that is what it pulled in. In this latest commit I pulled ghodss/yaml in.